missed 'postcss-scss' dependency in documentation

```
Cannot find module 'postcss-scss'
Require stack:
- /frontend/ember-cli-build.js
- /frontend/node_modules/ember-cli/lib/utilities/find-build-file.js
- /frontend/node_modules/ember-cli/lib/models/builder.js
- /frontend/node_modules/ember-cli/lib/tasks/build-watch.js
- /frontend/node_modules/ember-cli/lib/utilities/require-as-hash.js
- /frontend/node_modules/ember-cli/lib/cli/index.js
- /frontend/node_modules/ember-cli/bin/ember


Stack Trace and Error Report: /tmp/error.dump.da09687f5dc784333ce47232df1ace0b.log
```
run `yarn add -D postcss-scss` to solve

https://github.com/harisadam/ember-tailwind-postcss-scss/blob/fade9e63eed1200d7df29bcbcb9c0076650998fd/package.json#L50

--- a/README.md
+++ b/README.md
@@ -5,6 +5,7 @@
 - @csstools/postcss-sass  
 - tailwindcss  
 - postcss-import  
+- postcss-scss  
 
 ### ember-cli-build.js
 

--- a/README.md
+++ b/README.md
@@ -27,6 +27,9 @@ module.exports = function(defaults) {
       }
     }
   });
+
+  // ...
+};
 ```
 
 ### app.scss

